pkgs/by-name/em/emacsclient-commands/package.nix

```
@@ -0,0 +1,28 @@
+{ pkgs, lib, stdenv, fetchFromGitHub, buildGoModule, runCommandLocal }:
+
+buildGoModule rec {
+  pname = "emacsclient-commands";
+  version = "0.0.0-unstable-2023-09-22";
+
+  src = fetchFromGitHub {
+    owner = "szermatt";
+    repo = pname;
+    rev = "8f5c8a877794ed51f8225036e36fd5ce272b17f3";
+    hash = "sha256-OlcB5VqWYdl0wz1y8nmG6Xgdf5IPOUQ31UG1TDxQAis=";
+  };
+
+  vendorHash = "sha256-8oREed2Igz5UvUTDdOFwW5wQQy3H8Xj8epxo6gqnZFA=";
+
+  buildInputs = with pkgs; [go gnumake];
+
+  buildPhase = ''
+DESTDIR=$out/ make install
+'';
```

```suggestion
  installPhase = ''
    DESTDIR=$out/ make install
  '';
```
this should probably go into the installPhase and the formatting is a bit of.